api/shape/Arc.js

```diff
@@ -100,65 +95,67 @@ const reifyArc =
       }
     }
 
-    return spiral.absolute().tag(...plan.toGeometry().tags);
+    return spiral.absolute();
   };
 
-Shape.registerReifier('Arc', reifyArc(Z));
-Shape.registerReifier('ArcX', reifyArc(X));
-Shape.registerReifier('ArcY', reifyArc(Y));
-Shape.registerReifier('ArcZ', reifyArc(Z));
-
-const ArcOp = (type) => (x, y, z) => {
-  switch (type) {
-    case 'Arc':
-    case 'ArcZ':
-      if (x === undefined) {
-        x = 1;
-      }
-      if (y === undefined) {
-        y = x;
-      }
-      if (z === undefined) {
-        z = 0;
-      }
-      break;
-    case 'ArcX':
-      if (y === undefined) {
-        y = 1;
-      }
-      if (z === undefined) {
-        z = y;
-      }
-      if (x === undefined) {
-        x = 0;
-      }
-      break;
-    case 'ArcY':
-      if (x === undefined) {
-        x = 1;
-      }
-      if (z === undefined) {
-        z = x;
-      }
-      if (y === undefined) {
-        y = 0;
-      }
-      break;
-  }
-  const [c1, c2] = buildCorners(x, y, z);
-  return Shape.fromGeometry(taggedPlan({}, { type }))
-    .hasC1(...c1)
-    .hasC2(...c2);
-};
-
-export const Arc = ArcOp('Arc');
-export const ArcX = ArcOp('ArcX');
-export const ArcY = ArcOp('ArcY');
-export const ArcZ = ArcOp('ArcZ');
-
-Shape.prototype.Arc = Shape.shapeMethod(Arc);
-Shape.prototype.ArcX = Shape.shapeMethod(ArcX);
-Shape.prototype.ArcY = Shape.shapeMethod(ArcY);
-Shape.prototype.ArcZ = Shape.shapeMethod(ArcZ);
+const reifyArcZ = reifyArc(Z);
+const reifyArcX = reifyArc(X);
+const reifyArcY = reifyArc(Y);
+
+const ArcOp =
```

Comment:
Ah, it's mostly because I haven't cleaned up the constructor / reifier split properly from before.

Those should be able to be merged back together again.